Homework_12/Task_1.py

```diff
@@ -0,0 +1,24 @@
+from sympy import *
+x = Symbol("x")
+fx = x**3 - 50*x
+gx = -x**4 + 88*x**2 - 241
+fgx = fx - gx
+
+roots = solve(fgx)
+roots
```

Comment:
Почему такая запись? Это же не блокнот.
```suggestion
print(roots)
```